Pipfile

```diff
@@ -5,7 +5,7 @@ verify_ssl = true
 
 [dev-packages]
 flake8 = "*"
-pytest = "*"
+pytest = "==7.1.2"
 coverage = "*"
 pytest-cov = "*"
```

Comment:
may be worth explicitly pegging these to a version in a future PR

Reply:
I usually leave dev dependencies unpegged. I just pegged that one because pipenv kept installing 5.2.2 instead of the most recent version and I couldn't figure out why for a bit, and I forgot to unpeg it. That said, I don't have strong feelings one way or the other.